### ISSUE TITLE
net rpc shutdown command showing error message

### ISSUE THREAD

**Issue by aftermath8** [void-linux/void-packages#32761]
Hello everyone!

First, I want to thank the developers and the whole community for this very high quality distro!

So I chose Void to develop and host a network monitoring application. One of the components of this application is to shut down the remote machine (Windows based) in case an anomaly is detected, so I will use the following command;
`$ net rpc -I <IP> -U USER%PASSWD shutdown`

Here is the error I get:
```
open_policy failed: NT_STATUS_INVALID_HANDLE
open_policy failed: NT_STATUS_INVALID_HANDLE
```

Note that when I run a command from another server on the network it works without errors. 

Any help or advice would be very welcome as I really want to continue working with Void!

Thank y'all!

**Comment by aftermath8** [comment]
Anyone?

**Comment by ericonr** [comment]
Since you opened this issue in the wrong repo originally, you missed the issue template. Please include this information:

### System

* xuname:  
  *output of ``xuname`` (part of xtools)*
* package:  
  *affected package(s) including the version*: ``xbps-query -p pkgver <pkgname>``

**Comment by aftermath8** [comment]
Oh sorry, my bad!

So here we go:

- xuname: Void 5.13.13_1 x86_64 GenuineIntel notuptodate rFF
- package: `samba`

**Comment by aftermath8** [comment]
I'm sorry to relaunch the subject, but I'm in the final phase of development of the application and this error is the only grain of sand in the machine... 

It doesn't look like a network error. Plus, I ran the command with 3 different distributions (Kubuntu, Alpine Linux and OpenSUSE) and it worked fine. It's only with Void that it gets stuck. 

Some help would be really appreciated. Thanks!

**Comment by ericonr** [comment]
I want to try and do this locally, but haven't had the time. What debugging routes have you tried? Capturing an strace from it on Void and on another distro might help, I think. Does the machine you're controlling report anything about a failed attempt?

**Comment by lonwillett** [comment]
See PR https://github.com/void-linux/void-packages/pull/34916.

I suspect that it's some problem relating to its dependencies (talloc/tevent/tdb), although I'm by no means certain.

**Comment by ahesford** [comment]
Please test this against the (forthcoming) version 4.14.11.

**Comment by lonwillett** [comment]
Fixed for me with 4.14.11_1
